lib/src/layouts/yaru_navigation_rail_item.dart

```diff
@@ -14,20 +14,23 @@ enum YaruNavigationRailStyle {
 
 const _kSizeAnimationDuration = Duration(milliseconds: 200);
 const _kSelectedIconAnimationDuration = Duration(milliseconds: 250);
+const _kTooltipWaitDuration = Duration(milliseconds: 500);
 
 class YaruNavigationRailItem extends StatefulWidget {
   const YaruNavigationRailItem({
     super.key,
     this.selected,
     required this.icon,
     required this.label,
+    required this.tooltipMessage,
```

Comment:
Can we call it `tooltip` (see IconButton, FloatingActionButton, PopupMenuButton, ...) and please not make it required? Even though it makes sense to encourage users to set tooltips, we should allow them to build their own tooltips if they prefer. The reason could be as simple as that they want to use rich text, for example.

Reply:
Okay, you're right :)